lib/src/room.dart

```diff
@@ -332,8 +333,15 @@ class Room extends ChangeNotifier with ParticipantDelegate {
     delegate?.onDataReceived(participant, packet.payload);
   }
 
-  void _onTrackAdded(MediaStreamTrack track, MediaStream? stream, RTCRtpReceiver? receiver) {
+  void _onRemoteMuteChanged(String sid, bool mute) {
+    final track = localParticipant.tracks[sid];
+    //
+    // This will trigger signalClient.sendMuteTrack(sid, mute);
     //
+    track?.muted = mute;
```

Comment:
This will trigger `sendMuteTrack` back to server, which looks like the way JS SDK is implemented.
Is this intended and correct ?